tests/test_ckan_to_frictionless.py

```diff
@@ -15,11 +19,12 @@ class TestResourceConversion:
     '''
 
     def test_fixtures(self):
+        # Test simple conversion
         inpath = 'tests/fixtures/ckan_resource.json'
         exppath = 'tests/fixtures/frictionless_resource.json'
         indict = json.load(open(inpath))
         exp = json.load(open(exppath))
-        out = converter.resource(indict)
+        out = convert_c2f.resource(indict)
```

Comment:
I wouldn't have done this - this test of the conversion round trip should have been a separate test since it is neither c2f or f2c but both we are testing.

This would also have obviated a load of trivial renaming in the commit.